packages/studio/package.json

```diff
@@ -20,7 +20,7 @@
     "@minoru/react-dnd-treeview": "^3.4.1",
     "@restart/ui": "^1.5.2",
     "@vitejs/plugin-react": "^4.0.4",
-    "@yext/studio-plugin": "0.19.0",
+    "@yext/studio-plugin": "0.20.0",
```

Comment:
do we want to pin to a specific version, or can we add a caret here?

Reply:
I was thinking since we're releasing a lot of versions at the moment and some of them have had breaking changes, it might be a good idea to pin specific versions.

Reply:
Since we're going through a lot of versioning right now that might have a lot of unexpected changes, I was thinking it would be safer to pin versions.  I think this would prevent possible influx of issues from mixing and matching studio and plugin versions.  But I don't have too much experience with releases and versioning-- what are your thoughts?